src/styles/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__img {
+    width: 160px;
+    margin: 0 auto;
+    margin-top: 32px;
+  }
+
+  &__content {
+    padding: 0 16px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    color: $mainColor;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondaryColor;
+  }
+
+  &__rate {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__review {
+    color: $mainColor;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      color: $secondaryColor;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-cost {
+      color: $mainColor;
+      font-style: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+    width: 100%;
+    border-radius: 5px;
+    height: $linkHeight;
+    background-color: $linkColor;
+    border: unset;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+    cursor: pointer;
+    margin-bottom: 16px;
+  }
+
+  &__link:hover {
+    border: 1px solid $linkColor;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover

Reply:
This needs to be fixed too.